pkg/filter/dt/filter_http.go

```diff
@@ -139,23 +141,28 @@ type _httpFilter struct {
 	tccResourceInfoMap map[string]*TccResourceInfo
 }
 
+var _ proto.HttpPostFilter = (*_httpFilter)(nil)
+var _ proto.HttpPostFilter = (*_httpFilter)(nil)
+
 func (f *_httpFilter) GetKind() string {
 	return httpFilter
 }
 
-func (f _httpFilter) PreHandle(ctx *fasthttp.RequestCtx) error {
-	path := ctx.Request.RequestURI()
-	method := ctx.Method()
+func (f _httpFilter) PreHandle(ctx context.Context, fastHttpCtx *fasthttp.RequestCtx) error {
+	newCtx, span := tracing.GetTraceSpan(ctx, "http_filter_pre_handle")
```

Comment:
I recommend not adding `TraceContext` for this location. instead, add it to function `handleHttp1GlobalBegin`.